core/jazz_logs/config/test-config.json

```diff
@@ -1,5 +1,6 @@
 {
-  "BASE_URL": "https://{inst_elastic_search_hostname}/",
+  "BASE_URL": "{inst_elastic_search_hostname}/",
+  "KIBANA_URL": "{inst_kibana_search_hostname}",
```

Comment:
You will need to update [this](https://github.com/tmobile/jazz/blob/develop/builds/jazz-build-module/service-configuration-data-loader.groovy#L239) for the builds to work.

Reply:
yeah, already added, https://github.com/tmobile/jazz/pull/1087/files#diff-7c3ca5042104db9b883f027237d7004aR240

Reply:
Cool, I think I missed it.